ddtrace/contrib/pynamodb/patch.py

```diff
@@ -0,0 +1,86 @@
+"""
```

Comment:
We can format these files with `black` (`tox -e black`)

The lint checks can be run with `flake8` (`tox -e flake8`)